src/apps/backend/modules/vendor_account/internal/vendor_account_reader.py

```diff
@@ -33,3 +34,11 @@ def get_vendor_account_by_id(account_id: str, vendor_account_id: str) -> VendorA
             raise VendorAccountNotFoundError(vendor_account_id=vendor_account_id)
 
         return VendorAccountUtil.convert_vendor_account_db_to_vendor_account(vendor_account_db=vendor_account_db)
+
+    @staticmethod
+    def get_vendor_accounts_by_account_id(account_id: str) -> List[VendorAccount]:
+        vendor_accounts_db = VendorAccountRepository.collection().find(
```

Comment:
Whenever we return a list, it is best to define order by clause, in this cause, it maybe name, created_at